### PR TITLE
chore: Run fetch-cfa

### PR DESCRIPTION
**Pull request by nschonni**
Thanks for the contribution! **Below** is a template that can make it easier when submitting your Organization:

**Your Organization**: _ACME Agency_  
**GitHub Organization url**: _@acmeagency_  
**Country/Locality**: _Alberta, Canada_

Inclusion requirements:
- [ ] Referenced account is an [Organization](https://github.com/github/government.github.com#add-organization) not a User^
- [ ] If this is a government project, your Organization's description should include a reference to your country/agency.
- [ ] Make sure your Organization includes at least one public repository
- [ ] Please also include a URL for your organization that links back to your organization or agency homepage.
 
^ If you want to convert your account to an org, details can be found here: https://help.github.com/articles/converting-a-user-into-an-organization/
  
 :heart: GitHub Government Contributors

### CODE REVIEW

File: _data/civic_hackers.yml [hunk 1]
```diff
@@ -109,6 +121,7 @@ Civic Hackers:
   - opennorth-archive
   - openopps
   - openplans
+  - opensaltlake
```
Comment (mlinksva) [inline _data/civic_hackers.yml]: Failing CI. Looks like it's now codeforutah https://www.codeforutah.org/about

> Open Salt Lake no longer encompassed the core population of technologists in Utah nor the range of community work it should be embracing.
>
> Rebranding as Code for Utah

**Review by mlinksva — CHANGES_REQUESTED** [review]

File: _data/civic_hackers.yml (hunk 1 above)
Comment (nschonni) [reply _data/civic_hackers.yml]: Yeah, I'm trying to figure out why the script says it's OK, but the org doesn't exist

Comment (nschonni) [reply _data/civic_hackers.yml]: OK, I manually removed it, I guess that the script is pushing orgs out without checking if they're still valid

**Comment by mlinksva** [comment]
I think the entities I re-added in f2d9f41 should probably be added to https://github.com/codeforamerica/brigade-information/blob/master/organizations.json just noting in case I eventually do or someone else wishes to.

**Pull request merged**